src/_helpers/history.js

```diff
@@ -4,5 +4,5 @@ export const history = createBrowserHistory();
 
 export const logout = () => {
   localStorage.clear();
-  history.go("/");
+  window.location.href = "/";
```

Comment:
Color on this change

Reply:
history.go and history.push were not working as expected so that was the option that worked. we worked on this with Uluma